samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -387,6 +568,25 @@ static void button_handler_cb(uint32_t pressed, uint32_t changed)
 			prev_detect = k_uptime_get_32();
 		}
 	}
+
+	if (pressed & changed & BIT(1)) {
+		int err;
+		static int amb_light_idx;
+		struct sensor_value val;
+
+		dummy_ambient_light_value = dummy_amb_light_values[amb_light_idx++];
```

Comment:
Same thing here with modulo should be done at assignment time.

Reply:
Can not do it at assignment time as it is not a constant value, but it will never wrap around because it is reset on the line bellow.

Reply:
Oh would you look at that. I'll remember to bring my eyes to work next time